pkgs/swiftgen/example/avf_audio_wrapper.swift

```diff
@@ -0,0 +1,2867 @@
+
```

Comment:
Is this generated? If yes, add a header how to regenerate it.

Is this copied? If yes, does it need a license header and can it be in a third_party directory?

Reply:
It's complicated. It was generated by the version of swift2objc from the previous prototype of swiftgen, which had a few extra hacks to get it running. So the tools required to generate it won't be checked in yet. For now I'm thinking of it as hand written. So I guess I'll just add the license header.